docs/content/manual/pre-release/upgrade/test-node-drain-policy.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Test Node Drain Policy Setting
+---
+
+## With `node-drain-policy` is `block-if-contains-last-replica`
+
+### 1. Basic unit tests
+
+#### 1.1 Single worker node cluster with separate master node
+1.1.1 RWO volumes
+* Deploy Longhorn
+* Verify that there is no PDB for `csi-attacher`, `csi-provisioner`, `longhorn-admission-webhook`, and `longhorn-conversion-webhook`
+* Manually create a PVC (simulate the volume which has never been attached scenario)
+* Verify that there is no PDB for `csi-attacher`, `csi-provisioner`, `longhorn-admission-webhook`, and `longhorn-conversion-webhook` because there is no attached volume
+* Create a deployment that uses one RW0 Longhorn volume.
+* Verify that there is PDB for `csi-attacher`, `csi-provisioner`, `longhorn-admission-webhook`, and `longhorn-conversion-webhook`
+* Create another deployment that uses one RWO Longhorn volume. Scale down this deployment so that the volume is detached
+* Drain the node by `kubectl drain <node-name> --ignore-daemonsets --delete-emptydir-data --force`
+* Observe that the workload pods are evited first -> PDB of `csi-attacher`, `csi-provisioner`, `longhorn-admission-webhook`, and `longhorn-conversion-webhook` are removed -> `csi-attacher`, `csi-provisioner`, `longhorn-admission-webhook`, and `longhorn-conversion-webhook`, and instance-manager-e pods are evicted -> all volumes are successfully detached
+* Observe that instance-manager-r is NOT evicted.
+
+1.1.2 RWX volume
+* Deploy Longhorn
+* Verify that there is no PDB for `csi-attacher`, `csi-provisioner`, `longhorn-admission-webhook`, and `longhorn-conversion-webhook`
+* Create a deployment of 2 pods that uses one RWX Longhorn volume.
+* Verify that there is PDB for `csi-attacher`, `csi-provisioner`, `longhorn-admission-webhook`, and `longhorn-conversion-webhook`
+* Drain the node by `kubectl drain <node-name> --ignore-daemonsets --delete-emptydir-data --force`
+* Observe that the workload pods are evited first -> PDB of  `csi-attacher`, `csi-provisioner`, `longhorn-admission-webhook`, and `longhorn-conversion-webhook` are removed -> `csi-attacher`, `csi-provisioner`, `longhorn-admission-webhook`, and `longhorn-conversion-webhook`, and instance-manager-e pods are evicted -> all volumes are successfully detached
+* Observe that instance-manager-r is NOT evicted.
+
+#### 1.2 multi-node cluster
+1.2.1 Multiple healthy replicas
+* Deploy Longhorn
+* Verify that there is no PDB for `csi-attacher`, `csi-provisioner`, `longhorn-admission-webhook`, and `longhorn-conversion-webhook`
+* Manually create a PVC (simulate the volume which has never been attached scenario)
+* Verify that there is no PDB for `csi-attacher`, `csi-provisioner`, `longhorn-admission-webhook`, and `longhorn-conversion-webhook` because there is no attached volume
+* Create a deployment that uses one RW0 Longhorn volume.
+* Verify that there is PDB for `csi-attacher`, `csi-provisioner`, `longhorn-admission-webhook`, and `longhorn-conversion-webhook`
```

Comment:
According to https://github.com/longhorn/longhorn/issues/6026, the PDBs for `longhorn-admission-webhook` and `longhorn-conversion-webhook` do not exist after v1.5.x. I thought it would be great if additional info about it was added.

Reply:
Added the note. Thank you